bin.src/generateCoefficients.py

```diff
@@ -87,25 +87,33 @@
     else:
         tStart = args.tStart
 
+    # We have already ruled out the case where args.tSpan and args.tEnd are None, above.
```

Comment:
Right.  I forgot that tStart wasn't necessarily being handed in by the user.

Disregard my earlier comment.